internal/namespaces/ipfs/v1alpha1/ipfs_cli.go

```diff
@@ -40,6 +40,7 @@ func ipfsRoot() *core.Command {
 		Short:     `IPFS Pinning service API`,
 		Long:      `IPFS Pinning service API.`,
 		Namespace: "ipfs",
+		Groups:    []string{"labs"},
```

Comment:
Do not edit manually this file as it will create diff later on. I would suggest having a ipfs.GetCommands() function that modify the generated command to add this.